packages/core/package.json

```diff
@@ -9,6 +9,19 @@
     "effects",
     "types"
   ],
+  "exports": {
+    ".": {
+      "types": "./packages/redux-saga/index.d.ts",
```

Comment:
all of those paths need to be relative to this `package.json` but you have used some paths that are relative to the root of the monorepo. Remember that once we publish a package the monorepo "shell" won't be available - it's just an isolated package

Reply:
Right right, still getting used to a lerna / workspaces workflow for redux-saga, my apologies for the back and forth.

So I understand correctly, do I also need to update `packages/redux-saga` with entry points?  Right now its package.json file contains:

```
"main": "./dist/redux-saga-core-npm-proxy.cjs.js",
"module": "./dist/redux-saga-core-npm-proxy.esm.js",
"unpkg": "./dist/redux-saga.umd.min.js",
```